src/richie/plugins/glimpse/templates/richie/glimpse/glimpse.html

```diff
@@ -0,0 +1,38 @@
+{% load thumbnail %}{% spaceless %}
+
+{% with variant=variant|default:"card_square" %}
+
+    {% if instance.get_link %}
+        <a href="{{ instance.get_link }}"{% if instance.is_blank_target %} target="blank"{% endif %}
+        class="glimpse glimpse--{{ variant }}">
+    {% else %}
+        <div class="glimpse glimpse--{{ variant }}">
+    {% endif %}
+        {% if instance.image %}
+        <div class="glimpse__media">
+            <img src="{% thumbnail instance.image 200x200 crop upscale subject_location=instance.image.subject_location %}" alt="">
```

Comment:
Can you make it an srcset like we do elsewhere?
Also for the card variant, the image could be rectangular like on the course or program?